operators/api/v1alpha1/workspace_types.go

```diff
@@ -27,6 +28,9 @@ import (
 type WorkspaceSpec struct {
 	// The human-readable name of the Workspace.
 	PrettyName string `json:"prettyName"`
+
+	// The amount of resources associated with this workspace, and inherited by enrolled tenants.
+	Quotas WorkspaceResourceQuota `json:"quotas"`
```

Comment:
```suggestion
	Quota WorkspaceResourceQuota `json:"quota"`
```

The singular name is probably more appropriate (as also the Kubernetes resource is `ResourceQuota`. All occurrences should be updated.